dev/ci/ci-basic-overlay.sh

```diff
@@ -465,7 +468,11 @@ project coqtail "https://github.com/whonore/Coqtail" "main"
 ########################################################################
 # Deriving
 ########################################################################
-project deriving "https://github.com/arthuraa/deriving" "master"
+project deriving "https://github.com/proux01/deriving" "mc1343"
```

Comment:
this change should be in a separate PR from the jasmin (adding alg_tacs) change
also not sure we want to do this instead of setting allow_failure